proposals/nnnn-template-additional-submenus.md

```diff
@@ -0,0 +1,86 @@
+# Template Improvements: Additional Sub-Menus
+
+* Proposal: [SDL-NNNN](nnnn-template-additional-submenus.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC / HMI]
+
+## Introduction
+
+This proposal is to enable deeper menu drilling via AddSubMenu requests and greater control for OEMs over menu length and depth.
+
+## Motivation
+
+Currently, SDL is by far the most restrictive among its competitors for drilling down into menus. It only allows the top menu and one submenu (additional submenus can be "faked" using Choice Sets). CarPlay, for example, allows at least 3 levels of drilling, this was the most I could find in the apps I use (in addition to tabs at the top, which is a kind-of 4th level). CarPlay allows as much drilling as desired by the user in the car, but prevents scrolling while moving. SDL, on the other hand, reverses those restrictions, allowing as much scrolling as desired, while restricting drilling. It should be in the OEMs hands what restrictions are used.
+
+## Proposed solution
+
+### HMI Driver Distraction
+
+The proposed solution is a new set of restrictions that can be set by the OEM. This new set of restrictions should be the number of command items that are permissible in a list when the driver is distracted, and the second is the number of levels of submenus permitted when the driver is distracted. The default for scrolling items when distracted should default to 10, and number of levels of submenus will default to 3. This should be supported via the HMI, however, the author was unable to find a notification from Core to HMI of driver distraction. This would need to be provided, but then the HMI could handle hiding and showing list length and depth from there.
```

Comment:
"The default for scrolling items when distracted should default to 10, and number of levels of submenus will default to 3." seems a bit redundant.  Perhaps this could be updated to "The number of scrolling items when distracted should default to 10, and number of levels of submenus will default to 3."